Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.6.13-slim-buster
+
+RUN apt-get update && apt-get -y install --no-install-recommends \
+gcc \
+make \
+ca-certificates \
+wget \
+curl \
+telnet \
+net-tools \
+iputils-ping \
+dnsutils \
+git \
+gnupg \
+unzip \
+bzip2 && \
+apt-get clean && \
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN useradd --create-home --shell /bin/bash python
+USER python
+WORKDIR /home/python
+
+COPY . .
+
+RUN INSTALL_ON_LINUX=1 pip install -r requirements.txt
+RUN pip install git+https://github.com/DidierRLopes/TimeSeriesCrossValidation
+
+# is there a better way to do this? The chown is quite slow!
```

Comment:
When copying you can set the owner, see: https://github.com/moby/moby/pull/34263 for details

Reply:
Like this `COPY . . --chown=python:python`?

Reply:
I think it needs to be directly after the `COPY ` eg `COPY --chown=python:python . . `

Reply:
Nevermind, got it `COPY --chown=python:python . .`. Thanks for the tip.